sdk/textanalytics/Azure.AI.TextAnalytics/src/Transforms.cs

```diff
@@ -367,12 +367,16 @@ internal static AnalyzeHealthcareEntitiesResultCollection ConvertToAnalyzeHealth
             // Read entities.
             foreach (var document in results.Documents)
             {
+                BinaryData fhirBundle = document.FhirBundle.ValueKind is System.Text.Json.JsonValueKind.Object
+                    ? new BinaryData(document.FhirBundle)
+                    : null;
```

Comment:
May be better to throw here if it's not what we expect? Any other STJ method during deserialization would.

Reply:
What type of exception would you recommend? If I'm reading this right:
1. The old code would call `property.ThrowNonNullablePropertyIsNull()` if the property was `null`, which throws a `JsonException`, but only in Debug mode. 
2. Then, the old code calls `property.Value.EnumerateObject()`, which would throw an `InvalidOperationException` when `property.Value.ValueKind` is not of `ValueKind.Object` according to this [doc](https://learn.microsoft.com/en-us/dotnet/api/system.text.json.jsonelement.enumerateobject). This sounds very similar to what we want. Furthermore, I see that we have a few methods in this Transforms.cs file that also throw an `InvalidOperationException` in the case of similar discrepancies.

Based on this, I'm leaning towards throwing an `InvalidOperationException` . What do you think?

Reply:
Yes, I would throw an `InvalidOperationException`.